packages/babel-plugin-redux-saga/src/index.js

```diff
@@ -112,18 +76,21 @@ module.exports = function(babel) {
     FunctionDeclaration(path, state) {
       if (path.node.generator !== true) return
 
-      const functionName = path.node.id.name
-      const objectExtensionNode = getObjectExtensionNode(path, functionName)
-
+      var functionName = path.node.id.name
       var locationData = calcLocation(path.node.loc, state.file.opts.filename, state.opts.basePath)
 
-      var declaration = assignLoc(objectExtensionNode, locationData.fileName, locationData.lineNumber)
+      const extendedDeclaration = extendDeclarationWithLocation({
```

Comment:
technically this is also possible:
```js
const saga = function* saga() {}
```

and won't be visited with the plugin right now.

Reply:
I forgot that wanted to add FunctionExpression. Done.
from `error-demo` example
before:
![2018-03-29_13-04-48](https://user-images.githubusercontent.com/3198181/38085857-77a40416-3352-11e8-9bba-19a784787bbf.png)

after:
![2018-03-29_13-05-01](https://user-images.githubusercontent.com/3198181/38085847-6d8f66be-3352-11e8-812e-946b17c3d351.png)